crates/eww/src/config/eww_config.rs

```diff
@@ -24,11 +24,18 @@ pub struct EwwConfig {
     windows: HashMap<String, EwwWindowDefinition>,
     initial_variables: HashMap<VarName, DynVal>,
     script_vars: HashMap<VarName, ScriptVarDefinition>,
+    pub poll_var_trigger: HashMap<VarName, Vec<VarName>>,
 }
 
 impl Default for EwwConfig {
     fn default() -> Self {
-        Self { widgets: HashMap::new(), windows: HashMap::new(), initial_variables: HashMap::new(), script_vars: HashMap::new() }
+        Self {
+            widgets: HashMap::new(),
+            windows: HashMap::new(),
+            initial_variables: HashMap::new(),
+            script_vars: HashMap::new(),
+            poll_var_trigger: HashMap::new(),
```

Comment:
the name `poll_var_trigger` is reeaaaally confusing, see also my general comment on the name "trigger"

Reply:
the fact that "trigger" is singular here also really doesn't make sense. `poll_var_run_while_dependencies` or something would probably be a better name,... somewhat unweildly, but ehhh, it's a somewhat confusing edge-case so better be descript imo